src/lib/reasoners/sig.mli

```diff
@@ -128,11 +128,7 @@ module type SHOSTAK = sig
      [Some (t, false)], then there must be no context in which
      [solve r (fst X.make t)] raises [Unsolvable]. You have been warned! *)
 
-  (* choose the value to print and how to print it for the given term.
-     The second term is its representative. The list is its equivalence class
-  *)
-  val choose_adequate_model : Expr.t -> r -> (Expr.t * r) list -> r * string
-
+  val to_const_term : r -> Expr.t option
```

Comment:
Should be documented here.

Reply:
I put the documentation in the second signature. I do a copy.